torch/__init__.py

```diff
@@ -60,14 +50,9 @@ def _running_with_deploy() -> builtins.bool:
     USE_GLOBAL_DEPS,
     USE_RTLD_GLOBAL_WITH_LIBTORCH,
 )
+from torch.torch_version import __version__ as __version__
```

Comment:
```suggestion
from torch import version as version
from torch.torch_version import __version__ as __version__
```

Reply:
can you do this in a separate PR, I am not too sure how safe this is, and I don't want to keep running ci on this stack as it seems to be pretty good otherwise